### ISSUE TITLE
Indicators for A1.1: the protocol is open, free, and universally implementable

### ISSUE THREAD

**Issue by makxdekkers** [RDA-FAIR/FAIR-data-maturity-model-WG#20]
![image](https://user-images.githubusercontent.com/7534601/60017070-36c35380-9688-11e9-922e-9c99136ea6cd.png)

**Comment by makxdekkers** [comment]
Points raised in online meeting 3 on 18 June 2019

- The indicator should be split into two indicators. One about a free protocol and the other one about an open source protocol.
- An example is the Digital Object Interface Protocol (DOIP).
- Access only by telephone is a barrier to machine actionability.
- There may never be a machine-actionable protocol for access to some types of data, in which case a transparent phone-based protocol must be acceptable.

**Comment by RDA-FAIR** [comment]
![image](https://user-images.githubusercontent.com/47179213/60704251-2f137280-9f04-11e9-931d-b29e9171195e.png)

Here is a new version of the indicators!

**Comment by RDA-FAIR** [comment]
Please find the current version of the indicator(s) and their respective maturity levels for this FAIR principle. Indicators and maturity levels will be presented, as they stand, to the next working group meeting for approval.  In the meantime, any comments are still welcomed.

The editorial team will now concentrate on weighing and prioritizing these indicators. More information soon.

![image](https://user-images.githubusercontent.com/47179213/62354485-49de0480-b50d-11e9-8a70-6a69526ecd26.png)

**Comment by DrMaggie** [comment]
Since A1 itself refers to "(meta)data", it seems to me that there should probably also be two analogous indicators A1.1-01M and A1.1-02M that allow testing the protocol used for accessing the metadata. After all, the protocols could differ, e.g. depending on what repositories/catalogues are used for the metadata and the data...

**Comment by rwwh** [comment]
My extended interpretation of A1.1 also includes the protocol that researchers are required to follow to request access to restricted data. Such a protocol, even if it contains phone calls, should be properly documented.

For me, a protocol that says "The PI of our institute will decide whether you get access" is a lot less FAIR than a protocol that says "The consent specifies that the data can only be used for Lung Cancer Research. If you would like to get access to the data for your own Lung Cancer Study, fill in this application form that will be judged by the Data Access Committee of the institute"

**Comment by makxdekkers** [comment]
@rwwh How could we formulate an indicator for such a protocol (in the sense of the actions a person need to take)?
E.g.

**Actions to be taken by a reuser to get access to the data are well documented**

**Comment by keithjeffery** [comment]
I believe the intention behind A1.1 was machine processing protocol not human - but open to discuss

**Comment by bahimc** [comment]
Dear contributors,
 
Below you can find the indicators and their maturity levels in their current state as a result of the above discussions and workshops.
 
![image](https://user-images.githubusercontent.com/47179213/66299050-0f209000-e8f3-11e9-90e2-a76868fbc4ad.png)
 
**Please note that this thread is going to be closed**, within a short period of time.  The current state of the indicators, as of early October 2019, is now frozen, with the exception of the indicators for the principles that are concerned with ‘richness’ of metadata (F2 and R1). The current indicators will be used for the further steps of this WG, which are prioritisation and scoring.  Later on, they will be used in a testing phase where owners of evaluation approaches are going to be invited to compare their approaches (questionnaires, tools) against the indicators. The editorial team, in consultation with the Working Group, will define the best approach to test the indicators and evaluate their soundness. As such, the current set of indicators can be seen as an ‘alpha version’. In the first half of 2020, the indicators may be revised and improved, based on the results of the testing. **If you have any further comments, suggestions regarding that specific discussion, please share them with us. Besides, we invite you to have a look at the following two sets of issues.**
 
**Prioritisation**
 
[• Indicators prioritisation for Findability
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/30)[• Indicators prioritisation for Accessibility](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/31)
[• Indicators prioritisation for Interoperability
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/32)[• Indicators prioritisation for Reusability ](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/33)
 
**Scoring** 
 
[• Indicators for FAIRness | Scoring
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/34) 
We thank you for your valuable input!
